File: linklist/list.go
package linklist

type ListNode struct {
	Val  int
	Next *ListNode
}

func (l *ListNode) Array() []int {
	res := make([]int, 0, 5)
	for l != nil {
		res = append(res, l.Val)
		l = l.Next
	}
	return res
}

func MakeList(nums []int) *ListNode {
	var res *ListNode = new(ListNode)
	head := res
	for i := 0; i < len(nums); i++ {
		res.Next = new(ListNode)
		res = res.Next
		res.Val = nums[i]
	}
	return head.Next
}

File: linklist/list.py
from typing import List
class ListNode(object):
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next

def ListPrint(l:ListNode)->List:
    node=[]
    while(l):
        node.append(l.val)
        l=l.next
    return node 

def CreateList(num:List)->ListNode:
    cur=ListNode()
    res=cur
    for _,num in enumerate(num):
        cur.next=ListNode(num)
        cur=cur.next
    return res.next
